src/ts/core/HDiffPatch.ts

```diff
@@ -10,14 +10,14 @@ export default class HDiffPatch
         return new Promise(async (resolve) => {
             let result = await Neutralino.os.execCommand(`./public/hdiffpatch/hpatchz -f "${path.addSlashes(file)}" "${path.addSlashes(patch)}" "${path.addSlashes(output)}"`);
 
-            result = (result.stdOut ?? result.stdErr).includes('patch ok!');
+            const ret = (result.stdOut ?? result.stdErr).includes('patch ok!');
```

Comment:
normally result doesn't get cut short to "ret" but "res" instead as "ret" normally stands for return